hardware/raspberrypi/bcm2711_bootloader_config.md

```diff
@@ -32,6 +32,8 @@ Version: All
 
 If `1` and `WAKE_ON_GPIO=0` then `sudo halt` will switch off all PMIC outputs. This is lowest possible power state for halt but may cause problems with some HATs because 5V will still be on. GLOBAL_EN must be shorted to ground to boot.
 
+Pi 400 has a dedicated power button which operates even if the processor is switched off. This behaviour is enabled by default, however, `WAKE_ON_GPIO=2` may be set to fuse an external GPIO power button instead of the dedicated power button.
```

Comment:
Is "fuse" correct here, or is it a typo for "use"? :shrug:

Reply:
use